Cargo.toml

```diff
@@ -42,9 +42,9 @@ default-members = [
     "wgpu",
 ]
 
-[workspace.lints.clippy]
-manual_c_str_literals = "allow"
-ref_as_ptr = "warn"
+[workspace.lints.rust]
+manual_c_str_literals = "forbid"
+ref_as_ptr = "forbid"
```

Comment:
More generally, I don't think we should have anything ever set to `forbid` as it's too inflexible. `warn` is strong enough as we run with `-D warnings` in CI which fails if there are any warnings in the codebase.